src/styles/dialog-editor-boxes.scss

```diff
@@ -0,0 +1,151 @@
+.selected {
+  border: 3px solid $color-blue-300;
+}
+
+&.disabled {
+  color: $color-gray;
+  font-size: 18px;
```

Comment:
@dtaylor113 this is the conflicting line

Reply:
Hi,
I removed the conflicting line, please retest.
Thanks,
Dave